jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebElementRegistryImpl.java

```diff
@@ -41,6 +41,9 @@ public void registerLocator(String pageName, String elementName, By locator) {
         requireNonNull(elementName, "Parameter elementName is required.");
         requireNonNull(locator, "Parameter locator is required.");
         Map<String, By> elementRegistry = pageRegistry.computeIfAbsent(pageName, p -> new HashMap<>());
+        if (elementName.equals("@url")||elementName.equals("@title")){
```

Comment:
```suggestion
        if (elementName.equals("@url") || elementName.equals("@title")) {
```
Just formatting :slightly_smiling_face: